### PR TITLE
interactive list pull request 

### PR DESCRIPTION
**Pull request by theresetuyi**
## interactive list

-Use webpack to bundle JavaScript.
-Learn how to use proper ES6 syntax.
-Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,42 +1,90 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
```
Comment (Deepakdanger) [inline src/index.js]: - [ ] Kindly remove  & solve the linter manually.

File: src/modules/interactive.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-cycle */
```
Comment (Deepakdanger) [inline src/modules/interactive.js]: - [ ] Kindly remove  & solve the linter manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @theresetuyi , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/interactive.js (hunk 2 above)
Comment (Qoosim) [inline src/modules/interactive.js]: - [ ] This request has not been implemented. Kindly remove it as suggested by the previous reviewer.

File: src/index.js (hunk 1 above)
Comment (Qoosim) [inline src/index.js]: - [ ] This request has not been implemented. Kindly remove it as suggested by the previous reviewer.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @theresetuyi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/interactive.js (hunk 2 above)
Comment (alexander16108) [inline src/modules/interactive.js]: - [ ]  This request has not been implemented. Kindly remove it as suggested by the previous reviewer.

File: src/index.js [hunk 3]
```diff
@@ -1,42 +1,91 @@
 import './style.css';
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
```
Comment (alexander16108) [inline src/index.js]: - [ ] This request has not been implemented. Kindly remove it as suggested by the previous reviewer.

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @theresetuyi , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## To Highlight 💯

- No linter error ✔️
- Website looks professional 👍

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by DeliceLydia — APPROVED** [review]
## Hi @theresetuyi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**